pkg/util/virtualmachineinstance/virtualmachineinstance.go

```diff
@@ -47,6 +47,33 @@ func GetAllNonLiveMigratableVMINames(vmis []*kubevirtv1.VirtualMachineInstance,
 		if !migratable {
 			logrus.Infof("%s considered non-live migratable due to node affinities", vmiNamespacedName)
 			nonLiveMigratableVMINames = append(nonLiveMigratableVMINames, vmiNamespacedName)
+			continue
+		}
+
+		// container-disk
```

Comment:
There are similar functions on 
https://github.com/harvester/harvester/blob/3b9a3e36125de9b4c500703803bd7807e89a7bf5/pkg/controller/master/nodedrain/nodedrain_controller.go#L390

and 
https://github.com/harvester/harvester/blob/3b9a3e36125de9b4c500703803bd7807e89a7bf5/pkg/api/vm/handler.go#L411

Could we refactor those 3 places to a common funcion? thanks.

Reply:
Updated it. Thanks for the suggestion.